### ISSUE TITLE
PlatformIO IDE on Atom makes Atom crash on quit on Mac.

### ISSUE THREAD

**Issue by senceryazici** [platformio/platformio-atom-ide#1754]
When I try to quit via cmd+q on Mac atom quits without a problem if the platform ide package is disabled, when I enable it Atom crashes on quit and says, "Illegal instruction: 4". After I disable it and try to quit it says 

```
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: worker.js:71:22
Remote event names: destroyed, crashed
```
But it quits without a problem ? At least the Mac OS is happy about it. Otherwise I get the following (platform enabled)
<img width="439" alt="screen shot 2018-04-13 at 14 03 45" src="https://user-images.githubusercontent.com/15329726/38731570-89800284-3f23-11e8-8869-ef3a212e3e69.png">

I can share my settings about the package if you like. What is the problem ?

**Comment by zionlion77** [comment]
I have the same problem using Atom 1.27.2 and platformio-ide 2.1.4. With Platformio enabled Atom crashes upon quit (both via CMD+Q or menu item). Any idea how to solve this issue?

**Comment by zionlion77** [comment]
I just found out that I can prevent Atom from quitting unexpectedly when I have the PlatformIO home screen open. Maybe this helps.
Still, this bug is very annoying!

**Comment by ivankravets** [comment]
How to reproduce this issue?

**Comment by senceryazici** [comment]
To reproduce, I just enable the platformio on the settings panel, then if I try to quit the Atom editor, it crashes, no extra effort is needed to reproduce, it simply occurs when enabled.

**Comment by ivankravets** [comment]
Could you reproduce this issue with the latest version?

**Comment by ivankravets** [comment]
We have just released a new version [2.7.0](https://github.com/platformio/platformio-atom-ide/releases/tag/v2.7.0), please upgrade.

**Please note that we HIGHLY recommend moving to [PlatformIO IDE for VSCode](https://platformio.org/platformio-ide) which resolves tons of issues and brings a lot of new features.**
